api/src/main/java/org/openmrs/module/pihcore/setup/HtmlFormSetup.java

```diff
@@ -20,4 +29,26 @@ public static void setupHtmlFormEntryTagHandlers() throws Exception {
         htmlFormEntryService.addHandler(PihCoreConstants.HTMLFORMENTRY_FAMILY_HISTORY_RELATIVE_CHECKBOXES_TAG_NAME, new FamilyHistoryRelativeCheckboxesTagHandler());
     }
 
+    public static void loadHtmlForms() {
+
+        ResourceFactory resourceFactory = ResourceFactory.getInstance();
+        FormService formService = Context.getFormService();
+        HtmlFormEntryService htmlFormEntryService = Context.getService(HtmlFormEntryService.class);
+
+        try {
+            File htmlformDir = new File(PihCoreUtil.getFormDirectory());
+            Collection<File> files = FileUtils.listFiles(htmlformDir, null, true);
+
+            for (File file : files) {
+                try {
+                    HtmlFormUtil.getHtmlFormFromUiResource(resourceFactory, formService, htmlFormEntryService, "file:" + file.getAbsolutePath());
+                } catch (IOException e) {
+                    log.error("Unable to load HTML Form at path: " + file, e);
+                }
+            }
+        }
+        catch (Exception e) {
+            log.error("Unable to load HTML forms--this error is expected when running component tests");
```

Comment:
If it is not expected when you are _not_ running component tests, don't we want to throw the exception here?  Or maybe log the error and return a boolean indicating whether it succeeded or not so that calling code can determine how to handle this?